MANIFEST.in

```diff
@@ -1,15 +1,19 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 include *.rst
 include *.txt
-include *.py
 include buildout.cfg
 include tox.ini
 
-recursive-include src *
+exclude MANIFEST.in
```

Comment:
That is weird.  Why don't you want MANIFEST.in in the sdist??

Reply:
It is not needed there as it is only the template to generate the sdist.
We are dropping it in all projects of our company and never had a problem.

Why do you want to have it there?

Reply:
One should be able to re-create an sdist given just the sdist:
```
tar -xf some-sdist.tar.gz
cd some-sdist
python setup.py sdist
```

If configuration files like MANIFEST.in are missing, that's not possible.

Moreover, and I see this Linux distributions that package gevent, given an sdist, one should be able to do everything one can do given the source repository, including running tests and checking coverage. So that means sdists need to include coverage configuration, tox configuration—essentially everything that's not directly produced by the build process itself.

Reply:
I fixed this in zopefoundation/meta#35 by removing the exclude of `MANIFEST.in`.